atomicapp/external/templates/nulecule/Dockerfile.tpl

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.4.1
```

Comment:
This is bad as when a new release comes out (0.4.5 as a prime example) it still uses 0.4.5. We should be replacing this variable with the most current version of Atomic App as listed in `constants.py`